docs/UserGuide.md

```diff
@@ -21,28 +25,27 @@ SoConnect is a **desktop app for managing contacts and tasks**. It aims to help
 4. Double-click the file to start the app. A GUI similar to the one below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
-   Some example commands you can try:
+## Command Line Interface Tutorial
```

Comment:
Maybe we can give a disclaimer that this is for newbies and provide a skip link for CLI veterans.

Reply:
Ok